src/RepoAuditor/Plugins/GitHub/Impl/EnableRequirementImpl.py

```diff
@@ -79,6 +79,17 @@ def _EvaluateImpl(
         query_data: dict[str, Any],
         requirement_args: dict[str, Any],
     ) -> Requirement.EvaluateImplResult:
+        expected_value = self.enabled_by_default
+
+        # The value of the dynamic arg doesn't matter as much as whether it was provided
+        if requirement_args.get(self.dynamic_arg_name, False):

```

Comment:
I had a thought as I was walking in to work today. Perhaps a way to better communicate intent is `if self.dynamic_arg_name in requirement_args` (since the value itself doesn't matter).

Reply:
`requirement_args` will have the `self.dynamic_arg_name` key with either `True` or `False` (depending on the default). It is only in the test that we don't need to pass in the key-value pair.

So checking for presence does not actually check if the value is True or False.